src/Compilers/CSharp/Portable/Parser/CharacterInfo.cs

```diff
@@ -276,56 +242,22 @@ internal static bool ContainsDroppedIdentifierCharacters(string name)
 
         internal static bool IsLetterChar(UnicodeCategory cat)
```

Comment:
Please delete any APIs here that are now unused. I'd be surprised to see any uses of `IsCombiningChar`, for example.